doc/Analytics.xml

```diff
@@ -4208,6 +4208,128 @@ xmlns:acme="http://www.acme.com/schema"&gt;
+	  <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">Field [tt:Polygon]</para>
+            <para role="text">Graphical field definition in normalized coordinates.</para>
+          </listitem>
+		   <listitem>
+            <para role="param">TimeThreshold [xs:duration]</para>
+            <para role="text">Minimum duration the object should stay in the scene before it is judged to be abandoned.</para>
+          </listitem>
+		  <listitem>
+            <para role="param">Sensitivity [xs:float]</para>
+            <para role="text">This parameter allows to change the ratio between True and False detections. For "High" sensitivity value there are more True positive events as well as False positives.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Topic</term>
+          <listitem>
+            <para role="param">tns1:RuleEngine/ObjectAbandoned</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Source</term>
+          <listitem>
+            <para role="text">See <xref linkend="_msgSource"/></para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Data</term>
+          <listitem>            
+            <para role="param">ObjectId [tt:StringList]</para>
+            <para role="text">List of objects triggering this rule.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>      
+      <para>See below example for a Object Abandoned definition:</para>
+      <programlisting><![CDATA[<tt:RuleDescription Name="tt:ObjectAbandoned">
+  <tt:Parameters>
+    <tt:ElementItemDescription Name="Field" Type="tt:Polygon"/>
+    <tt:SimpleItemDescription Name="TimeThreshold" Type="xs:duration"/>
+    <tt:SimpleItemDescription Name="Sensitivity" Type="xs:float"/>
+  </tt:Parameters>
+  <tt:Messages IsProperty="true">
+    <tt:Source>
+      ...
+    </tt:Source>
+    <tt:Data>      
+      <tt:SimpleItemDescription Name="ObjectId" Type="tt:StringList"/>
+    </tt:Data >
+    <tt:ParentTopic>
+      tns1:RuleEngine/ObjectAbandoned
+    </tt:ParentTopic>
+  </tt:Messages>
+</tt:RuleDescription>
+]]></programlisting>
+    </section>
+	<section>
+      <title>Object Removed</title>
+      <para>The Object Removed rule is defined by a simple non-intersecting polygon as an area of interest and removed time threshold interval. Object Removed module continuously monitors a scene and detects and alarms when any valuable property is removed from the scene.</para>
+      <para>The Object Removed module is used for loss prevention and securing valuable items in government, industrial, retail, corporate facilities, and public places such as museums and national landmarks.</para>
```

Comment:
Instead of "is used for", suggestion to change the sentence like below,

"For example, object removed rule can be used in .."

Reply:
Updated PR as per Sujith's suggestion.